unassigned_issues/unassigned_issues_scraper.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from django.http import HttpResponse
+from gci.config import get_api_key
+from IGitt.GitHub import GitHubToken
+from IGitt.GitHub.GitHubRepository import GitHubRepository
+
+from community.git import get_org_name
+
+
+def run(mr_requests):
+    issues_number_list = []
+    for pr in mr_requests:
+        if pr.state._name_ == 'OPEN':
+            for commit in pr.commits:
+                mystatus = commit.combined_status._name_
+                break
+            if mystatus == 'SUCCESS' or mystatus == 'PENDING':
+                for issue in pr.closes_issues:
+                    if str(pr.number) != issue.number:
+                        if str(issue.state) == 'open':
+                            if len(issue.assignees) is 0:
+                                issues_number_list.append(issue.number)
+    return issues_number_list
+
+
+def unassigned_issues_activity_json(request):
+    try:
+        GH_TOKEN = get_api_key('GH')
+    except Exception:
+        return HttpResponse('[]')
+    org_name = get_org_name()
+    repo_name = org_name + '/' + org_name
```

Comment:
For current preview using `netlify `  and to prevent CI breakage, we have to go through this way if user want he/she just have to change it to like this.

`` repo_name = org_name + '/' + org_repo_name``

Reply:
I will add a one line comment there so that anyone can understand.

Reply:
>Why to check only one repo? Why not for all repos in a org? 

I have read it wrongly ,sorry :+1: 
@Vamshi99  IMO it may cause `Rate Limit` Problem, not totally sure.

Reply:
hmm, I also don't know the API limit. @jayvdb does it cross the API limit if it checks for all repos?

Reply:
Ya we hit the API limit quite quickly.